db/migrate/20230303175550_remove_unique_index.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUniqueIndex < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :email
+    add_column :users, :email, :string, default: '', null: false
+  end
+end
```

Comment:
- [ ] Good job tweaking your migrations and adding the users table, however, this migration is not working:

![image](https://user-images.githubusercontent.com/57421823/224163739-d41168c6-6b94-4cce-90cd-f060a9325e73.png)


![image](https://user-images.githubusercontent.com/57421823/224162739-c05cbb84-5387-4124-b117-6f6299e0bb37.png)

The reason is that you are trying to remove a column that does not exist in line 3.

Solution: Just remove that line, and reset the database: `rails db:reset`.  This will drop the database, create it again and run the corrected migrations.

![image](https://user-images.githubusercontent.com/57421823/224163172-8f4edd34-58ec-4a46-9738-1fcbe4649077.png)